.editorconfig

```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: 
```

Comment:
The EditorConfig looks interesting! It is new to me, but seems like a great way to improve consistency within a certain project.

However, it is not part of implementing `Math.Enum.mode`, so I would prefer it if you could provide it in a separate pull request.

Reply:
Whoops, I can rebase this branch to remove this. 

See #14

Reply:
As previously noted #14 has the commits to add `.editorconfig` to the project.

Thanks to `git rebase -i` I've removed the offending commits from this branch.